scripts/create-icons.js

```diff
@@ -19,7 +19,7 @@ async function generateIcons() {
   const icnsOutPath = path.join(buildDir, 'icon.icns');
 
   if (!fs.existsSync(pngPath)) {
-    console.warn("WARNING: No icon.png found in your webhapp's UI assets.");
+    console.warn("WARNING: No icon.png found. If you're using the systray option, an icon.png (1024x1024 pixel) is required to be provided at the root level of your webhapp's UI assets.");
```

Comment:
For me it had to be 256x256 minimum.
```suggestion
    console.warn("WARNING: No icon.png found. If you're using the systray option, an icon (256x256 pixel) is required at /resources/ui/icon.png");
```

Reply:
I changed the resolution as suggested. But the other part about the icon having to be in `/resources/ui/icon.png` might be misleading. The `/resources/ui` folder is the unpacked content of the ui.zip of .webhapp file that needs to be added to the pouch folder. It gets unpacked into the `/resources/ui` folder by the `yarn pouch:unpack` command, so if the icon.png is part of the webhapp file then it lands in there automatically. And if you put the icon into the `ui/resources` folder independently it will get removed again with the next invocation of `yarn pouch:unpack`.

Reply:
Ah, got it.